### ISSUE TITLE
Dota 2 Battle Pass Bonus Week Chest Bugged. 

### ISSUE THREAD

**Issue by PutDatCookieDown** [ValveSoftware/Dota2-Gameplay#4611]
### Description

The multiplier on the battle pass bonus week chests is bugged. I truly appreciate the free battle pass, don't get me wrong, I'm very thankful. But, it is very annoying to see that the only way I can gain a massive amount of battle pass points, after grinding BP quests for a couple of days, is bugged and not working. Whenever I start kicking the bonus chests the score goes up to 8K points then after a slight delay it drops down to 1K points. Not only once has that happened. It happened a couple of times, and it is very frustrating.

**Comment by HeavensounD** [comment]
This has happened to me as well on multiple occasions. I also get an error modal right after I claim the Bonus Week Chest. Hope this gets fixed.

**Comment by seke626** [comment]
This has happened to me twice now. Very frustrating.

**Comment by PutDatCookieDown** [comment]
Yes, it is very frustrating. I don't think the are going to fix it any time soon. Seems like it's been around for a while now.

> This has happened to me twice now. Very frustrating.

**Comment by TheArtOfJoking** [comment]
BRUH 5-6 of my chests gave me only 1 kk and 4 of them went down from 2-4 k to 1k
